### ISSUE TITLE
What to focus on for the workshop

### ISSUE THREAD

**Issue by tischi** [tischi/i2k-2020-s3-zarr-workshop#11]
@joshmoore 
cc @constantinpape 
Since weI can now open the multiscale ome.zarr files from s3 I was wondering what would make sense to work on next, for the workshop.

What we have on `s3` is this:

```
images.json
a.xml
b.xml
c.xml
a.zarr
b.zarr
c.zarr
```

What I currently achieved is to open an `a.zarr` on s3 using a local `a.xml` stored on a file system.

I wonder what to do next. 

My feeling is that directly opening an `a.zarr` stored on s3 with bdv _without_ using the xml might be something, because then people could potentially already during the workshop write their own `my.zarr`, store in in some object store, and view it in Fiji using a menu entry `Plugins > MoBIE > Open OME.Zarr.S3...` which would let them enter such stuff...
```
      <Key>my.zarr</Key>
      <SigningRegion>us-west-2</SigningRegion>
      <ServiceEndpoint>https://s3.embl.de</ServiceEndpoint>
      <BucketName>i2k</BucketName>
      <Authentication>Anonymous</Authentication>
```
...and which would open the file.

Is it within scope to let users write their own multiscale zarr files to an object store?

Alternatives are, I focus on
- opening `a.xml` in the object store
    - this allows loading metadata mapping from voxel space to physical space, using an affine transform
- opening `images.json` in the object store
    - this would allow showing a whole multi-scale data set, with fancy coloured (interactive) label masks

What do you think?

**Comment by joshmoore** [comment]
Sorry for the slow response. (Never hesitate to re-ping)

> My feeling is that directly opening an a.zarr stored on s3 with bdv without using the xml might be something

This sounds good, but is similar to the n5.n5 approach, no?

> Is it within scope to let users write their own multiscale zarr files to an object store?

Yes. I can make that happen. For smallish datasets, https://play.minio.io:9000/minio/ might be an option as well.

For the other two alternatives, I can't really say. My impulse is always to try to get a specification in place first 🙂 

cc: @will-moore

**Comment by tischi** [comment]
> For the other two alternatives, I can't really say. My impulse is always to try to get a specification in place first 

I interpret this as not "teaching" the students how to open the xml and json files, because this could create the wrong impression that this is some sort of standard? At least this is how I feel.

**Comment by joshmoore** [comment]
Yes. Unless we use it as a lesson in what might happen naturally because it's cool but then requires effort to give up. Like 🚬 .

**Comment by tischi** [comment]
@joshmoore 
Ok, I'll be working on reading the zarr directly, however this will only work for files within one channel and one timepoint for now. So we would need to use an example like this for the workshop, ok?!

**Comment by joshmoore** [comment]
http://blog.openmicroscopy.org/file-formats/community/2020/11/04/zarr-data/ lists a few from idr0083, idr0079, and idr0053 if that's any help.
